utils/check_repo.py

```diff
@@ -663,6 +668,7 @@ def find_all_documented_objects():
     "logger",  # Internal logger
     "logging",  # External module
     "requires_backends",  # Internal function
+    "AltCLIPVisionModel",  # Internal module
```

Comment:
I doubt this should be here. Could you double check? (probably it should be `AltRobertModel`??)

Reply:
I add it because the module is being imported by main init and every model in init should be documented while we don't want it to be documented so I add it.

Reply:
I didn't add altroberta because it do not imported in main init.

Reply:
This is related one of the above comment we have. It's just a misunderstanding of Sylvain's comment.

`AltCLIPVisionModel` is meant to be in main init, and should be documented.
So we should remove it from the list here. Thank you :-)